src/condor_utils/file_transfer.cpp

```diff
@@ -4668,14 +4706,10 @@ FileTransfer::computeFileList(
 			// While (* total_bytes_ptr) and numFiles should both be 0
 			// at this point, we should probably be explicit.
 			filesize_t logTCPStats = 0;
-			return ExitDoUpload( & logTCPStats,
-				/* num files */ 0,
-				s, saved_priv, protocolState.socket_default_crypto,
-				/* upload success */ false,
-				/* do upload ACK (required to avoid hanging the shadow and starter */ true,
-				/* do download ACK */ false,
-				/* try again */ false,
-				holdCode, holdSubCode, holdReason.c_str(), __LINE__ );
+			UploadExitInfo xfer_info;
+			xfer_info.setError(holdReason, holdCode, holdSubCode);
+			xfer_info.doAck(TransferAck::UPLOAD).line(__LINE__);
```

Comment:
See previous question.

Reply:
There a few other places further down with the same construction, which I didn't explicitly call out.